nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/log/LogTimeCellFactory.java

```diff
@@ -0,0 +1,26 @@
+package org.nmrfx.processor.gui.log;
+
+import javafx.scene.control.TableCell;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Formatter for LocalDateTime table cells in a LogRecord table
+ */
+public class LogTimeCellFactory extends TableCell<LogRecord, LocalDateTime> {
```

Comment:
It is strange to have a `SomethingFactory` extends `Something`. Is it really a factory?

Reply:
I thought it was because I was using it to set the cellFactoryProperty but looking closer at the example in the docs they just called their class SomethingFormat. I will rename to LogTimeCellFormatter.